snapshot/snapshot.go

```diff
@@ -82,6 +84,19 @@ func NewSnapshotter(ctx context.Context, cfg *config.SnapshotterConfig) (snapsho
 		return nil, errors.Wrap(err, "parse recover policy")
 	}
 
+	cgroupConfig, err := config.ParseCgroupConfig(cfg.DaemonConfig.CgroupConfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "parse cgroup configuration")
+	}
+	log.L.Infof("parsed cgroup config: %#v", cgroupConfig)
+	cgroupMgr, err := cgroup.NewManager(cgroup.Opt{
+		Name:   "nydusd.slice",
```

Comment:
`.slice` is a systemd style that I don't think we have to follow

Reply:
Fixed.